README.md

```diff
@@ -6,27 +6,45 @@ The derived reactors customize this base class for particular MuJoCo model files
 
 ## Prerequisites
 
-MuJoCo depends on [GLFW](https://www.glfw.org), a graphics library that you must install.  On macOS:
+
+### Linux (Ubuntu)
+
+1. Install GLFW, a graphics library used by MuJoCo
 
 ```sh
-brew install glfw
+apt install libglfw3-dev
 ```
 
-MuJoCo itself seems to be best installed from source.  The following worked for me on macOS:
+2. Download pre-built Mujoco v3.2.6 binaries from [here](https://github.com/google-deepmind/mujoco/releases/tag/3.2.6).
+```sh
+wget wget https://github.com/google-deepmind/mujoco/releases/download/3.2.6/mujoco-3.2.6-linux-x86_64.tar.gz
+```
 
+3. Install to `/opt` 
 ```sh
-git clone git@github.com:google-deepmind/mujoco.git
-cd mujoco
-mkdir build
-cd build
-cmake .. -DCMAKE_INSTALL_PREFIX=/usr/local
-cmake --build .
-sudo cmake --install .
+sudo mkdir -p /opt/mujoco-3.2.6-linux-x86_64/
+tar xvf mujoco-3.2.6-linux-x86_64.tar.gz --directory /opt/mujoco-3.2.6-linux-x86_64/
 ```
+### macOS
+
+MuJoCo depends on [GLFW](https://www.glfw.org), a graphics library that you must install. On macOS:
+
+```sh
+brew install glfw
+```
+
+2. Download the Mujoco v3.2.6 DMG disk image from [here](https://github.com/google-deepmind/mujoco/releases/tag/3.2.6).
+```sh
+wget https://github.com/google-deepmind/mujoco/releases/download/3.2.6/mujoco-3.2.6-macos-universal2.dmg
+```
+
+3. Install to `/Application` by double-clicking the DMG and dragging MuJoCo.app into `/Applications` 
```

Comment:
Could you verify that this works?

Reply:
That was the first technique I tried on macOS, and I was unable to make it work.  That's why I went with installing from source.

Reply:
I see, I tried it out on my old MacBook and it wasn't that easy. Will adjust the docs